samples/multicluster-razee/README.md

```diff
@@ -0,0 +1,481 @@
+    kubectl expose deployment my-notebook --port=8888 --target-port=8888
+    ```
+
+2. Create a port-forward to communicate with JupyterLab:
+
+    ```bash
+    kubectl port-forward svc/my-notebook 8888:8888 &
+    ```
+
+3. Open your browser and go to [http://localhost:8888/](http://localhost:8888/).
+
+4. Create a new notebook in the server
+
+## Create a `FybrikApplication` resource for the notebook
+
+Create a [`FybrikApplication`](../reference/crds.md#fybrikapplication) resource to register the notebook workload to the control plane of Fybrik: 
+
+```yaml
+kubectl config use-context kind-control
+
+cat <<EOF | kubectl apply -f -
+apiVersion: app.fybrik.io/v1alpha1
+kind: FybrikApplication
+metadata:
+  name: my-notebook
+  labels:
+    app: my-notebook
+spec:
+  selector:
+    clusterName: kind-control
+    workloadSelector:
+      matchLabels:
+        app: my-notebook
+  appInfo:
+    intent: Fraud Detection
+  data:
+    - dataSetID: "fybrik-notebook-sample/paysim-csv"
+      requirements:
+        interface: 
+          protocol: fybrik-arrow-flight
+EOF
+```
+
+Notice that:
+
+* The `selector/clusterName` field specifies the name of cluster which our Jupyter notebook workload is running on.
+* The `selector/workloadSelector` field matches the labels of our Jupyter notebook workload.
+* The `data` field includes a `dataSetID` that matches the asset identifier in the catalog.
+* The `protocol` indicates that the developer wants to consume the data using Apache Arrow Flight. For some protocols a `dataformat` can be specified as well (e.g., `s3` protocol and `parquet` format).
+
+Run the following command to wait until the `FybrikApplication` is ready:
+
+```bash
+while [[ $(kubectl get fybrikapplication my-notebook -o 'jsonpath={.status.ready}') != "true" ]]; do echo "waiting for FybrikApplication" && sleep 5; done
+while [[ $(kubectl get fybrikapplication my-notebook -o 'jsonpath={.status.assetStates.fybrik-notebook-sample/paysim-csv.conditions[?(@.type == "Ready")].status}') != "True" ]]; do echo "waiting for fybrik-notebook-sample/paysim-csv asset" && sleep 5; done
+```
+
+## Read the dataset from the notebook
+
+In your **terminal**, run the following command to print the endpoint to use for reading the data. It fetches the code from the `FybrikApplication` resource:
+
+```bash
+kubectl config use-context kind-control
+ENDPOINT_SCHEME=$(kubectl get fybrikapplication my-notebook -o jsonpath={.status.assetStates.fybrik-notebook-sample/paysim-csv.endpoint.fybrik-arrow-flight.scheme})
+ENDPOINT_HOSTNAME=$(kubectl get fybrikapplication my-notebook -o jsonpath={.status.assetStates.fybrik-notebook-sample/paysim-csv.endpoint.fybrik-arrow-flight.hostname})
+ENDPOINT_PORT=$(kubectl get fybrikapplication my-notebook -o jsonpath={.status.assetStates.fybrik-notebook-sample/paysim-csv.endpoint.fybrik-arrow-flight.port})
+printf "${ENDPOINT_SCHEME}://${ENDPOINT_HOSTNAME}:${ENDPOINT_PORT}"
+```
```

Comment:
suggest to use `printf "${ENDPOINT_SCHEME}://${ENDPOINT_HOSTNAME}:${ENDPOINT_PORT}\n"` (just add a new line) and made the dataset endpoint much more easy to read and copy.

Reply:
It's a good idea. Done. Thanks.